### PR TITLE
Vimeo fixes

### PR DESCRIPTION
**Pull request by stephencelis**
Should further reduce bandwidth.

### CODE REVIEW

File: Sources/PointFree/Account/PrivateRss.swift [hunk 1]
```diff
@@ -288,7 +288,7 @@ private func item(forUser user: User, episode: Episode) -> RssItem {
     enclosure: .init(
       length: episode.fullVideo.bytesLength,
       type: "video/mp4",
-      url: episode.fullVideo.downloadUrl
+      url: episode.fullVideo.downloadUrl(.hd720)
```
Comment (stephencelis) [inline Sources/PointFree/Account/PrivateRss.swift:291]: We could potentially do 1080p for the latest episode...but maybe we wait till we figure out if our bandwidth gets in the acceptable range.

File: Sources/PointFree/Account/PrivateRss.swift [hunk 2]
```diff
@@ -374,7 +374,7 @@ private func invalidatedItem(errorMessage: String) -> RssItem {
     enclosure: .init(
       length: Episode.ep0_introduction.fullVideo.bytesLength,
       type: "video/mp4",
-      url: Episode.ep0_introduction.fullVideo.downloadUrl
+      url: Episode.ep0_introduction.fullVideo.downloadUrl(.sd540)
```
Comment (stephencelis) [inline Sources/PointFree/Account/PrivateRss.swift:377]: For the invalid feed I'm just doing standard def.

File: Sources/PointFree/AtomFeed.swift [hunk 3]
```diff
@@ -74,6 +74,7 @@ private func items() -> [RssItem] {
   return Current
     .episodes()
     .sorted(by: their({ $0.freeSince ?? $0.publishedAt }, >))
+    .prefix(4)
```
Comment (stephencelis) [inline Sources/PointFree/AtomFeed.swift:77]: This will only put the latest 4 episodes in the public feed.

File: Sources/PointFree/AtomFeed.swift [hunk 4]
```diff
@@ -130,7 +131,7 @@ can access your private podcast feed by visiting \(url(to: .account(.index))).
     return .init(
       length: video.bytesLength,
       type: "video/mp4",
-      url: video.downloadUrl
+      url: video.downloadUrl(.sd540)
```
Comment (stephencelis) [inline Sources/PointFree/AtomFeed.swift:134]: Standard def for public feed.

File: Sources/Models/Episode.swift [hunk 5]
```diff
@@ -364,30 +364,17 @@ public struct Episode: Equatable {
...
     }
     public var streamingSource: String {
       "https://player.vimeo.com/video/\(self.vimeoId)?pip=1"
     }
 
-    public init(
-      bytesLength: Int,
-      downloadUrl: String,
-      streamingSource: String
-    ) {
```
Comment (stephencelis) [inline Sources/Models/Episode.swift:383]: Got rid of this with a regex find and replace.

**Review by mbrandonw — APPROVED** [review]

**Pull request merged**